custom/icds_reports/management/commands/bihar_data_pull.py

```diff
@@ -0,0 +1,80 @@
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+import csv
+
+from custom.icds_reports.models.aggregate import AwcLocation
+
+headers = [
+	'StateID',
+	'StateName'
+	'DistrictID',
+	'DistrictName',
+	'ProjectID',
+	'ProjectName',
+	'SectorID',
+	'SectorName',
+	'AWCID',
+	'AWCName',
+	'Name',
+	'Phone_Number'
+	'Has Bank Account?',
+	'Bank IFSC Code',
+	'Bank Account Number',
+	'Bank Name',
+	'Bank branch name']
+
+data_rows = [headers]
+
+def get_person_case(case):
+	for parent in case.get_parent():
+		if parent.get_case_property('person')=='person':
+			return parent
+
+
+def fetch_case_properties(case, awc):
+	return [
+    	awc['state_site_code'],
+    	awc['state_name'],
+    	awc['district_site_code'],
+    	awc['district_name'],
+    	awc['block_site_code'],
+    	awc['block_name']
+        awc['supervisor_site_code'],
+    	awc['supervisor_name'],
+        awc['awc_site_code'],
+        awc['awc_name'],
+        get_person_case.get_case_property('name'),
```

Comment:
`get_person_case` is a function. so it would value. it should be `get_person_case()`. Moreover it would be better and faster if you just pass the person case object as an argument to this function so that its not evaluated twice